BlueprintUICommonControls/Tests/Sources/PerformancePlayground.swift

```diff
@@ -219,6 +219,61 @@ class PerformancePlayground: XCTestCase {
         }
     }
 
+    func test_increment_state_changes() {
```

Comment:
I think it's worth waiting for #389 to land, and then re-using `LabeledBox` for an apples to apples comparison with test_deep_and_wide_grid.